### PR TITLE
About Status messages

### PR DESCRIPTION
**Pull request by jorge-campo**
- 125-0 first draft (dev. questions pending)
- 125-0 first draft ready for SME review
- 125-0 first draft ready for SME review-FIX
- 125-0 first draft ready for SSME review-FIX2

### CODE REVIEW

**Comment by jorge-campo** [comment]
Notes for the reviewer:
 
- The content for review is docs/en/using-status/about-status-messages.md.
- I'm working on a new FAQ article with answers to common questions, where I'll discuss how messages history and retention work.
- I include a PDF of how the content looks in Status Help, so you can see the table and illustration in context.
[About Status messages - Status Help.pdf](https://github.com/status-im/help.status.im/files/9402256/About.Status.messages.-.Status.Help.pdf)

File: docs/en/using-status/about-status-messages.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+![The peer-to-peer network sends messages to every node and doesn't rely on a central server.](./about-status-messages/125-0-1-dark.png#only-light)
+![The peer-to-peer network sends messages to every node and doesn't rely on a central server.](./about-status-messages/125-0-1-light.png#only-dark)
+
+## Built-in privacy and security
+
+The Status app secures your messages the entire time they are in transit using end-to-end encryption. This industry standard protection uses strong encryption keys and ensures that only you and the recipients can read your messages. No one else (not even Status) can read your conversations or identify you or other participants.
+
+When you communicate with someone on the Status app, your messages, attachments, sender metadata, group chats, and group metadata are all end-to-end encrypted.
+
+Status incorporates the [Perfect Forward Secrecy :octicons-tab-external-16:][perfect-foward-secrecy]{:target="_blank"} (PFS) encryption mechanism, ensuring that encryption keys change on every message. If your keys are compromised, only the associated message is compromised. All previous and future messages remain private.
```
Comment (jrainville) [inline docs/en/using-status/about-status-messages.md:51]: Is that true cc @richard-ramos ?

**Review by jrainville — COMMENTED** [review]
Looks good. 
I would just wait for someone like Richard to look at the PFS encryption mechanism since it's the first time I hear about it.

File: docs/en/using-status/about-status-messages.md (hunk 1 above)
Comment (jorge-campo) [reply docs/en/using-status/about-status-messages.md:51]: I believe @cammellos knows about this and can (hopefully) have a quick look at this content 🙏

Comment (cammellos) [reply docs/en/using-status/about-status-messages.md:51]: That's correct, 
though not completely accurate

```
All previous and future messages remain private.
```

PFS covers only previous messages, future messages are not covered by PFS, that's a different property (post compromise security/backward privacy), and what we use has a good post compromise security, but it's not watertight.

Comment (jorge-campo) [reply docs/en/using-status/about-status-messages.md:51]: Thanks, @cammellos!

**Pull request merged**